crates/stackable-versioned-macros/src/attrs/common/item.rs

```diff
@@ -317,6 +318,10 @@ fn default_default_fn() -> SpannedValue<Path> {
     )
 }
 
+// TODO (@Techassi): Add validation for when from_name AND from_type or both
+// none => is this action needed in the first place?
```

Comment:
I assume this was supposed to be:

```suggestion
// TODO (@Techassi): Add validation for when from_name AND from_type are both
// none => is this action needed in the first place?
```